src/phasorpy/phasor.py

```diff
@@ -2831,18 +2836,37 @@ def phasor_threshold(
         modulation_max = numpy.nan
     else:
         threshold_mean_only = False
-    if fill_value is None:
-        fill_value = numpy.nan
 
     if threshold_mean_only is None:
         return _phasor_threshold_nan(mean, real, imag, **kwargs)
 
     if threshold_mean_only:
-        return _phasor_threshold_mean(
-            mean, real, imag, mean_min, mean_max, fill_value, **kwargs
+        if open_interval:
+            return _phasor_threshold_mean_open(
+                mean, real, imag, mean_min, mean_max, **kwargs
+            )
+        return _phasor_threshold_mean_closed(
+            mean, real, imag, mean_min, mean_max, **kwargs
         )
 
-    return _phasor_threshold(
+    if open_interval:
+        return _phasor_threshold_open(
+            mean,
+            real,
+            imag,
+            mean_min,
+            mean_max,
+            real_min,
+            real_max,
+            imag_min,
+            imag_max,
+            phase_min,
+            phase_max,
+            modulation_min,
+            modulation_max,
+            **kwargs,
+        )
+    return _phasor_threshold_closed(
```

Comment:
In case you want to shorten this:
```Python
func = _phasor_threshold_open if open_interval else _phasor_threshold_closed
func(
    mean,
    real,
    imag,
    mean_min,
    ...
)
```